AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -88,6 +90,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ * An ASLayoutSpec subclass that can wrap a ASLayoutable and calculates the layout of the child.
+ */
+@interface ASWrapperLayoutSpec : ASLayoutSpec
```

Comment:
Look into my comment. TLDR: Yes we could get remove layoutSpecThatFits: and introduce something that would allow to return an `id<ASLayoutable>`, but as this change is already pretty big I don't want to add that for now and we could follow up with a PR. It should be not to hard with the new architecture.